unblob/file_utils.py

```diff
@@ -12,6 +13,24 @@ def round_up(size: int, alignment: int):
     return alignment * math.ceil(size / alignment)
 
 
+def find_first(file: io.BufferedIOBase, pattern: bytes) -> int:
```

Comment:
I'd recommend making the chunk_size an optional parameter with default value set 0x1000, making the function more generic as we might want to re-use it in the future for other formats but need to read by smaller or larger increment.